### DISCUSSION TITLE
Allow branch update with rebase in mobile app

### DISCUSSION THREAD

**Discussion by skalahonza** [community/community#17542] [Mobile] [answered]
In a browser, one can update branch with rebase. But the mobile does not show the rebase option and thus only merge can be performed.

**Comment by HenSquared** [comment, marked as answer]
Hi @skalahonza,

Thanks a bunch for leaving this feedback (and others for upvoting). I've opened up an issue internally to track this feature suggestion and will have some discussion with the team. If we have any updates for you or questions, we'll reach back out here. 

Thanks!

**Reply by Maligurgah** [reply]
21700957

**Comment by derekperkins** [comment]
This was one of many reasons I ditched the mobile app. It is a long ways off from feature parity with mobile web

**Comment by ccronemberger** [comment]
It would be nice to also make it clear that it is not possible to be done from the mobile app. I spent a long time trying to find it, so if we had a message saying that this is not supported we could save more people from wasting their time looking for a feature that has no reason to not be there.

**Comment by Dan1grl** [comment]
Rebase is an interesting project, just wondering what happened with the goggle app to download if you do not have IOS...Overall I have much to learn and is very intrigued!

**Comment by olantwin** [comment]
This is still an issue 4 years later. It would be great if this could be addressed, as it's the one missing piece of PR review on mobile.
